### ISSUE TITLE
`po dfu open` not working

### ISSUE THREAD

**Issue by shivams** [nrobinson2000/po#21]
On Ubuntu 20.04, I have installed `po-util` as well as `dfu-util`. However, `po dfu open` doesn't work anymore. It used to work in previous Ubuntu versions. `po dfu close` works however. Do I need to install something else as well to make it work?

**Comment by nrobinson2000** [comment]
Hey @shivams, sorry you are experiencing this issue.

On Linux, `po dfu open` depends on [custom-baud](https://github.com/nrobinson2000/po/tree/master/depend/custom-baud), a C program that mimics the behavior of [stty](https://linux.die.net/man/1/stty) to change the baud rate of the device to induce DFU mode. Additionally, `po serial open` depends on custom-baud to induce listening mode.

If `po serial open` is failing as well, then custom-baud possibly failed to compile during the installation process.

Can you verify that the `~/.po-util/bin/custom-baud` directory exists and that you can recompile and use custom-baud?

```bash
cd ~/.po-util/bin/custom-baud
make
# Put device into DFU mode
./custom-baud /dev/ttyACM0 14400
stty -F /dev/ttyACM0 9600
```

**Comment by shivams** [comment]
Thank you @nrobinson2000 for your prompt response. 

I just just tried compiling it with `make` but it was showing error: `custom-baud.c:24:10: fatal error: stropts.h: No such file or directory`. Turns out, `stropts.h` is not present in Ubuntu 20.04 when you install `libc6-dev` (https://askubuntu.com/questions/1237091/stropts-h-install-for-ubuntu-20-01)

So, I removed the inclusion of `stropts.h` from `custom-baud.c` and then it compiled fine. Then, I ran your `./custom-baud /dev/ttyACM0 14400` command and, voila, it opened the DFU mode on my device :smiley: And now, `po dfu open` also works fine. 

I think the issue is resolved, but we should introduce a permanent fix for this.
